gui/wxpython/vdigit/wxdigit.py

```diff
@@ -26,22 +26,152 @@
 @author Martin Landa <landa.martin gmail.com>
 """
 
-import grass.script.core as grass
-
-from grass.pydispatch.signal import Signal
+import os
+import sys
+from ctypes import byref, c_double, c_int, create_string_buffer, pointer
 
-from core.gcmd import GError
+import grass.script.core as grass
 from core.debug import Debug
+from core.gcmd import GError
 from core.settings import UserSettings
-from vdigit.wxdisplay import DisplayDriver, GetLastError
 
 try:
-    from grass.lib.gis import *
-    from grass.lib.vector import *
-    from grass.lib.vedit import *
-    from grass.lib.dbmi import *
+    from grass.lib.dbmi import (
+        DB_C_TYPE_STRING,
+        DB_NEXT,
+        DB_OK,
+        DB_SEQUENTIAL,
+        db_append_string,
+        db_close_database_shutdown_driver,
+        db_convert_column_value_to_string,
+        db_execute_immediate,
+        db_fetch,
+        db_get_column_name,
+        db_get_column_sqltype,
+        db_get_column_value,
+        db_get_cursor_table,
+        db_get_string,
+        db_get_table_column,
+        db_get_table_number_of_columns,
+        db_init_handle,
+        db_init_string,
+        db_open_database,
+        db_open_select_cursor,
+        db_set_handle,
+        db_set_string,
+        db_shutdown_driver,
+        db_sqltype_to_Ctype,
+        db_start_driver,
+        db_test_value_isnull,
+        dbCursor,
+        dbHandle,
+        dbString,
+    )
+    from grass.lib.gis import (
+        GMAPSET_MAX,
+        GNAME_MAX,
+        G_find_vector2,
+        G_free,
+        G_name_is_fully_qualified,
+    )
+    from grass.lib.vector import (
+        GV_AREA,
+        GV_BOUNDARY,
+        GV_CENTROID,
+        GV_LINE,
+        GV_LINES,
+        GV_POINT,
+        GV_POINTS,
+        GV_TOPO_PSEUDO,
+        WITHOUT_Z,
+        Map_info,
+        Vect_append_point,
+        Vect_area_alive,
+        Vect_area_perimeter,
```

Comment:
This is a mistake in the code, the function `Vect_area_perimeter` doesn't exist, instead use
```suggestion
        Vect_get_area_perimeter,
```

and also please fix the call of the function